i18n/locales/en.ts

```diff
@@ -2969,16 +2980,18 @@ Stack Hint: To spend before the secret is revealed, Vanderpoole uses his signatu
       title: `Make another payment`,
       nav_title: `Make another payment`,
       heading_one: `Make another payment`,
-      paragraph_one: `The night is still young, and you're not flying your Budgetcopter home. Got time for another beer? Now that you and Laszlo have your asymmetrical transactions and a flow worked out with revocation keys, let's run the protocol a few more times.`,
-      paragraph_two: `The two transactions representing the current state are on the screen, where you have paid Laszlo 1,000 satoshis. Send another 1,000 satoshi payment to Laszlo.`,
-      list_one: 'Update the amounts and output scripts',
+      paragraph_one: `The night is still young, and you're not flying your Budgetcopter home. Got time for another beer? Now that you and Laszlo have your asymmetric transactions and a flow worked out with revocation keys, let's run the protocol again.`,
+      paragraph_two: `The two commitment transactions representing the current state where you have paid Laszlo 1,000 sats for a beer are on your ePhone Infinity screen.`,
+      heading_two: `Instructions`,
+      list_one: `Update the amounts and output scripts for Laszlo's new commitment transaction`,
       list_two: "Sign Laszlo's transaction and send it to him",
-      list_three: `Laszlo will revoke his last state by sending you <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base whitespace-nowrap">REVOCATION_LASZLO_2</span>`,
+      list_three: `Update your commitment transaction: Laszlo will revoke his last state by sending you <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base whitespace-nowrap">REVOCATION_LASZLO_1</span>. Update the amounts and output scripts in your commitment transaction.`,
       list_four: 'Send your transaction to Laszlo so he can sign it',
-      list_five: `Once you have Laszlo's signature, send him your <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base whitespace-nowrap">REVOCATION_YOU_3</span> `,
+      list_five: `Once you have Laszlo's signature, send him your previous revocation key, <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base"> REVOCATION_YOU_2 </span> `,
```

Comment:
same here should it be 3 or 2?

Reply:
It should be `REVOCATION_YOU_2`. The copy on here is describing the act of sending the revocation keys for the previous commitment transactions, which invalidates them and makes the new set of commitment transactions in this exercise the only ones.

I think you're looking for copy that tells the user about the new set of revocation keys, `REVOCATION_YOU_3` and `REVOCATION_LASZLO_2` which would be an improvement since those are the revocation keys you need to update the commitment transactions with. Unfortunately, that would require a larger rework of these instructions. Based on my track record with this kind of stuff, a larger change to this would not be approved before the deadline we are trying to hit for chapter 10 so I'd prefer to tackle it after we release.